package.json

```diff
@@ -58,14 +58,19 @@
     "@babel/preset-env": "^7.4.5",
     "babel-eslint": "^10.0.1",
     "babel-loader": "^8.0.6",
+    "css-loader": "^3.2.0",
     "eslint": "^6.5.1",
     "eslint-loader": "^3.0.2",
     "eslint-plugin-react": "^7.16.0",
     "forever": "^1.0.0",
     "nodemon": "^1.19.1",
     "npm-run-all": "^4.1.5",
+    "optimize-css-assets-webpack-plugin": "^5.0.3",
     "pino-pretty": "^3.2.0",
     "source-map-loader": "^0.2.4",
+    "style-loader": "^1.0.0",
+    "terser-webpack-plugin": "^2.2.1",
+    "url-loader": "^2.2.0",
```

Comment:
What was the reason for adding these at the top level package.json?

Reply:
Moved them into userbase-js for clarity